k8s/openebs-operator.yaml

```diff
@@ -409,3 +409,84 @@ spec:
         hostPath:
           path: /var/openebs/sparse
 ---
+apiVersion: v1
```

Comment:
generic comment: Should we have any label that identifies these as openebs

Reply:
added `openebs.io/component: admission-server`

Reply:
Not sure what this label means.. should we good to have version related label? However, going ahead with the merge.